README.md

```diff
@@ -68,89 +120,36 @@ Following must be available locally:
-    * ```setx TM_SMTP_HOST "smtp-server-host-here"```
-    * ```setx TM_SMTP_PORT "smtp-server-port-here"```
-    * ```setx TM_SMTP_USER "smtp-server-user-here"```
-    * ```setx TM_SMTP_PASSWORD "smtp-server-password-here"```
-
-### Creating the DB
-We use [Flask-Migrate](https://flask-migrate.readthedocs.io/en/latest/) to create the database from the migrations directory.  If you can't access an existing DB refer to DevOps page to [set up a local DB in Docker](https://github.com/hotosm/tasking-manager/wiki/Dev-Ops#creating-a-local-postgis-database-with-docker) create the database as follows:
+**Tests**
+The project includes a suite of Unit and Integration tests that you should run after any changes
 
 ```
-python manage.py db upgrade
+python -m unittest discover tests/server
 ```
 
-### Running Locally
+## Create the Database from existing data
+We use [Flask-Migrate](https://flask-migrate.readthedocs.io/en/latest/) to create the database from the migrations directory. If you can't access an existing DB refer to DevOps page to [set up a local DB in Docker](https://github.com/hotosm/tasking-manager/wiki/Dev-Ops#creating-a-local-postgis-database-with-docker) create the database as follows
 
-#### API Development only
-If you plan to only work on the API you don't need to build the client and can run as follows:
+```
+python3 manage.py db upgrade
+```
 
-* Run the server:
-    * ``` python manage.py runserver -d -r```
-* Point your browser to:
-    * [http://localhost:5000/api-docs](http://localhost:5000/api-docs)
+## API
 
-#### Seeing the client
-If you want to see the client you will need to follow all the instruction in **Client Development** section then build the client as follows:
+If you plan to only work on the API you only have to build the server architecture. Install the server dependencies, and run these commands:
 
-* Build the client using gulp:
-    * ```cd client```
-    * ```gulp build```
-* You can now run the server as above from the root dir:
-    * ```cd ..```
+* Run the server:
     * ``` python manage.py runserver -d -r```
 * Point your browser to:
-    * [http://localhost:5000](http://localhost:5000)
-
-### Running Unit Tests
-The project includes a suite of Unit and Integration tests that you should run after any changes
-
-```
-python -m unittest discover tests/server
-```
+    * [http://localhost:5000/api-docs](http://localhost:5000/api-docs)
 
-## <a name="DevOps">Dev Ops</a>
+## DevOps
 If you encounter any issues while setting up a dev environment, please visit our [FAQ ❓ page](https://github.com/hotosm/tasking-manager/wiki/Dev-Environment-FAQ) to find possible solutions.
 
+## Localisation
 
-### Server Config
-
-#### Environment Vars
-
-On boot the Tasking Manager App will look for the following environment vars:
+Please see the [Localisation Wiki](https://github.com/hotosm/tasking-manager/wiki/Localisation) to translate the tasking Manager to your language.
```

Comment:
I would change this link to the Transifex repository instead (https://www.transifex.com/hotosm/tasking-manager-3/dashboard/) as that's where we do all the translations.